crates/squawk_parser/tests/snapshots/tests__select_ok.snap

```diff
@@ -1,6 +1,7 @@
 ---
 source: crates/squawk_parser/tests/tests.rs
 input_file: crates/squawk_parser/tests/data/ok/select.sql
+snapshot_kind: text
```

Comment:
I think if you upgrade `cargo insta` to a more recent version it stops including the `snapshot_kind` attr, but not 100% sure

When I run these locally it removes this attribute